### PR TITLE
chore: Remove Makefile in favour of package.json scripts

### PR DESCRIPTION
**Pull request by chr1sjf0x**
Per discussions about all commands using `yarn [command]`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,10 @@
     "migrate": "yarn workspaces foreach run migrate",
     "codegen": "yarn workspaces foreach run codegen",
     "format": "yarn workspaces foreach run format",
-    "test": "yarn workspaces foreach run test"
+    "test": "yarn workspaces foreach run test",
+    "docker": "docker-compose build",
+    "db": "docker compose up db --wait & docker compose exec db ./reset.sh & yarn migrate & yarn codegen",
```
Comment (stephenh) [inline package.json:12]: Question: What's the behavior of `&` vs. `&&` which is usually what I type out of habit?

**Review by stephenh — APPROVED** [review]
Nice!

File: package.json (hunk 1 above)
Comment (chr1sjf0x) [reply package.json]: Good eye, the single & means run the command in the background. && gives the serial execution we desire here 👍

**Comment by stephenh** [comment]
Sweet, thnks @chr1sjf0x !

**Pull request merged**